maker/src/trading/orderbook_client.rs

```diff
@@ -12,7 +12,10 @@ pub struct OrderbookClient {
 impl OrderbookClient {
     pub fn new() -> Self {
         Self {
-            client: reqwest::Client::new(),
+            client: reqwest::Client::builder()
+                .timeout(std::time::Duration::from_secs(10))
```

Comment:
🤔 is 10 seconds conservative enough? e.g. the generation of the fake scid takes sometimes more than 10 seconds. how about 30 seconds to be on the safe side?

Reply:
> 🤔 is 10 seconds conservative enough? e.g. I haven't stopped the time but the generation of the fake scid takes sometimes very long. how about 30 seconds to be on the safe side?

Sure, 10s was arbitrary! Shall we put 30s everywhere to be safe? or were you just talking about this place?

Reply:
Yes, let's go for 30 seconds to be safe. 😊